sdk/documentintelligence/ai-document-intelligence-rest/README.md

```diff
@@ -0,0 +1,209 @@
+# Azure DocumentIntelligence (formerly FormRecognizer) REST client library for JavaScript
```

Comment:
I notice that the changelog file is empty, could you prepare that?

I guess we may fail to release in ci if we have no relevant version entry in changelog.

Reply:
I didn't mean to skip them, I just didn't get to them 😅